src/docs/blog/coursier_migration.md

```diff
@@ -1,14 +1,18 @@
 # Twitter's Coursier Migration
 
-## Meta Info
+## Dates
 
-Author: Twitter Engineering Effectiveness - Build
+Twitter's Coursier migration happened during Q1 2018.
 
-Abstract: This blog post discusses the motivation, preparation work, deployment processes, and results of replacing Ivy with Coursier in Pants.
+This post is published in August 2018.
```

Comment:
No need to spell out "This post is published in...". Can just include `August 31, 2018`... people will know what that means =P